sparkle/platform/settings_help.py

```diff
@@ -78,6 +78,8 @@ class Settings:
     DEFAULT_config_budget_per_run = 600
     DEFAULT_config_number_of_runs = 25
 
+    DEFAULT_portfolio_construction_timeout = 172000
+
```

Comment:
Also needs a setting parse so user can change this default value from the sparkle_settings.ini